common/app/model/meta.scala

```diff
@@ -80,39 +56,14 @@ final case class Commercial(
   tags: Tags,
   metadata: MetaData,
   isInappropriateForSponsorship: Boolean,
-  sponsorshipTag: Option[Tag],
-  isFoundationSupported: Boolean,
-  isAdvertisementFeature: Boolean,
-  hasMultipleSponsors: Boolean,
-  hasMultipleFeatureAdvertisers: Boolean,
   hasInlineMerchandise: Boolean
 ) {
 
-  def sponsorshipType: Option[String] = {
-    if (isSponsored(None)) {
-      Option("sponsoredfeatures")
-    } else if (isAdvertisementFeature) {
-      Option("advertisement-features")
-    } else if (isFoundationSupported) {
-      Option("foundation-features")
-    } else {
-      None
-    }
-  }
-
-  def isSponsored(maybeEdition: Option[Edition]): Boolean =
-    DfpAgent.isSponsored(tags.tags, Some(metadata.sectionId), maybeEdition)
-
-  def needsHighMerchandisingSlot(edition:Edition): Boolean = {
-    DfpAgent.isTargetedByHighMerch(metadata.adUnitSuffix,tags.tags,edition,metadata.url)
+  def needsHighMerchandisingSlot(edition: Edition): Boolean = {
+    DfpAgent.isTargetedByHighMerch(metadata.adUnitSuffix, tags.tags, edition, metadata.url)
   }
-
-
-  def javascriptConfig: Map[String, JsValue] = Map(
-    ("isAdvertisementFeature", JsBoolean(isAdvertisementFeature))
```

Comment:
I assume that there isn't anything that continues to rely on this flag client-side?